EXILED/Exiled.API/Features/BanManager.cs

```diff
@@ -0,0 +1,62 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="BanManager.cs" company="Exiled Team">
```

Comment:
ExMod Team (iirc)

Reply:
yeah that normal until the PR who modify it get merge